README.md

```diff
@@ -142,14 +142,14 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
       python3 -m venv --upgrade-deps venv
       source venv/bin/activate
       pip cache remove llama_cpp_python
-      pip install 'instructlab[rocm]' \
-         --extra-index-url https://download.pytorch.org/whl/rocm6.0 \
-         -C cmake.args="-DLLAMA_HIPBLAS=on" \
-         -C cmake.args="-DAMDGPU_TARGETS=all" \
-         -C cmake.args="-DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang" \
-         -C cmake.args="-DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++" \
-         -C cmake.args="-DCMAKE_PREFIX_PATH=/opt/rocm" \
-         -C cmake.args="-DLLAMA_NATIVE=off"
+      CMAKE_ARGS="-DLLAMA_HIPBLAS=on \
+         -DAMDGPU_TARGETS=all \
+         -DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang \
+         -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++ \
+         -DCMAKE_PREFIX_PATH=/opt/rocm \
+         -DLLAMA_NATIVE=off" \
+         pip install 'instructlab[rocm]' \
+         --extra-index-url https://download.pytorch.org/whl/rocm6.0
```

Comment:
Currently, we use ROCM 6.1.2, shouldn't the index be `rocm6.1`?

Reply:
@ivanbaldo could you make this update? once that's in and this is rebased I think we can get it in!

Reply:
@ivanbaldo pinging in case you still want to get this in!

Reply:
I didn't change the index from the original, if the index needs to be changed, I suggest a different PR for that specific change just in case.

Reply:
Fine w/ me - just needs to be rebased

Reply:
Thanks for the reviews!
I have rebased it now.